### ISSUE TITLE
"Unable to locate publishable resources" On install

### ISSUE THREAD

**Issue by OmarMonterrey** [munafio/chatify#148]
Hello, Im getting that "Unable to locate publishable resources" Whenever I try to install using chatify:install. Also I noticed that even with the provider properly registered it's not listed when I use "php artisan vendor:publish"
![providers](https://user-images.githubusercontent.com/7505415/137599722-70eb5e10-1af8-4f09-99cb-65bd07fb5475.png)
![ChatifyError](https://user-images.githubusercontent.com/7505415/137599723-2bbdf730-e644-4f3b-974e-37e75e094e5f.png)

Thanks in advanced

**Comment by munafio** [comment]
@OmarMonterrey Hi,
You either have to register Chatify service provider into `app.js` and you can follow the official docs:
chatify.munafio.com > Installation > Step 5

Or try running the following command:
```
php artisan vendor:publish --provider="Chatify\ChatifyServiceProvider"
```
or with `tag`, to publish a specific asset
```
php artisan vendor:publish --provider="Chatify\ChatifyServiceProvider" --tag="chatify-config"
```

**Comment by OmarMonterrey** [comment]
Hello @munafio , thanks for your answer. I tried with those commands (Provider was registered) but it didn't worked because the publishable elements wasn't there. I figured out that the command "PHP" wasnt working as "CLI" but as "cgi-fcgi", that's because I tried to install it directly on the shared hosting of my customer.
Because of that, this conditional was never meet so the resources weren't publishable (ChatifyServiceProvider.php:36)
```
if ($this->app->runningInConsole()) {
            $this->commands([
                InstallCommand::class,
                PublishCommand::class,
            ]);
            $this->setPublishes();
        }
```
So I changed
`if ($this->app->runningInConsole())`
with
`if ($this->app->runningInConsole() || 1)`

Published and then reverted it (To prevent future issues) and it worked :)
